app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -391,9 +414,10 @@ main#page-outcomes-report
           label.edit-label.editing-only(v-if="editing" for="startDate") &nbsp; (edit)
 
     .org-results(v-if="org && !loading")
-      outcomes-report-result-component(:org="org" v-bind:editing="editing" :showLicense="showLicense" :showLicenseSummary="kind !== 'student'")
+      outcomes-report-result-component(:org="org" :other-org="otherOrg" v-bind:editing="editing" :showLicense="showLicense" :showLicenseSummary="kind !== 'student'" :showOther="showOther")
       if includeSubOrgs
         outcomes-report-result-component.sub-org(v-for="subOrg, index in subOrgs" v-bind:index="index" v-bind:key="subOrg.kind + '-' + subOrg._id" v-bind:org="subOrg" v-bind:editing="editing" v-bind:isSubOrg="true" v-bind:parentOrgKind="org.kind")
+        outcomes-report-result-component.sub-org(v-if="showOther" v-for="subOrg, index in otherSubOrgs" v-bind:index="index" v-bind:key="subOrg.kind + '-' + subOrg._id" v-bind:org="subOrg" v-bind:editing="editing" v-bind:isSubOrg="true" v-bind:parentOrgKind="org.kind")
```

Comment:
I see this warning
<img width="709" alt="Screenshot 2024-03-15 at 12 52 32 PM" src="https://github.com/codecombat/codecombat/assets/6376157/3757a3bb-ecf9-4f52-8167-63271be3c69d">


Also, can we combine things and show in subOrg so that coco and oz school stats make only a single entry for school in the report?

Reply:
😂 can we do that later? since it's quite hard to merge org objects (so that I pass the otherOrg into component and do merge there). every org objects shapes different 😹

Reply:
can u separate the keys atleast to remove this warning?

Reply:
it's separated now. the oz school now contains `other-` prefix so the warning would go away .